components/engine_traits/src/checkpoint.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::Result;
+
+pub trait Checkpoint {
```

Comment:
It should be named `Checkpointable` just like `Writeable`.

Reply:
Done